types/lhr/lhr.d.ts

```diff
@@ -6,11 +6,14 @@
 
 import {Result as AuditResult} from './audit-result';
 import {ConfigSettings} from './settings';
+import Gatherer from '../gatherer'
```

Comment:
I _believe_ this will break now, after #12914. `GatherMode` will have to be defined in here (or one of the lhr/ `d.ts` files) and then `gatherer.d.ts` can import it and `export import GatherMode = GatherMode_` or whatever if we want to maintain existing `LH.Gatherer.GatherMode` references